src/dnx.coreclr.unix/tpa.cpp

```diff
@@ -0,0 +1,105 @@
+// TODO: Unify this file with the one generate at build time for
+// dnx.coreclr on windows.
+
+#include "stdafx.h"
+#include "tpa.h"
+
+BOOL CreateTpaBase(LPWSTR** ppNames, size_t* pcNames, bool bNative)
+{
+    const size_t count = 35;
+    LPWSTR* pArray = new LPWSTR[count];
+
+    if (bNative)
+    {
+        pArray[0] = strdup("Internal.Runtime.Uri.ni.dll");
```

Comment:
I'll defer this feedback.  I want to unify this file with the existing generated version (but I will need to do a LPWSTR -> LPTSTR conversion first). Because of that, I don't want to make major changes.